NEWS

```diff
@@ -1,3 +1,14 @@
+28-Jun-2019 IGNITION v0.33.0
+
+  Bug Fixes:
+
+    - Fix regression where Ignition would panic if no the guestinfo.* keys
```

Comment:
This doesn't read right, maybe drop the `no`?

Reply:
Done, yeah that was a typo.